src/js/vre/chapter36/config/form.js

```diff
@@ -32,60 +32,46 @@ const formConfig = {
   title: 'Apply for vocational counseling',
   subTitle: 'Form 28-8832',
   defaultDefinitions: {
-    fullName
   },
   chapters: {
     applicantInformation: {
       title: 'Applicant Information',
+      initialData: {
+        socialSecurityNumber: '424242424'
+      },
```

Comment:
Got this from the cookbook but couldn't get it to work.

Reply:
initialData should be a property on the specific page. [Here](https://github.com/department-of-veterans-affairs/vets-website/blob/7b33713a457c0f47175f244fbb0a22f93d4dfe18/src/js/edu-benefits/1995/config/form.js#L88) is an example. The documentation for this is inconsistent, thanks for pointing it out.